### DISCUSSION TITLE
Use vlc media player in android studio instead of exoplayer

### DISCUSSION THREAD

**Discussion by tim-vk** [jellyfin/jellyfin-androidtv#1730] [General]
Hi, So I'd thought to take a shot at adding a bit to this project: However I'm running against a silly thing: When building & running the app in android studio the app seems to use Exoplayer by default. As far as I understand VLC should be the new (non-deprecated) player. How do I get android studio to use vlc player instead?

Thanks!

**Comment by tim-vk** [comment]
I found the answer myself! (i'm a fool!)

For future noobs like me: 
The preferred player can be selected within the app itself.
Settings -> playback -> preferred media player

Hope this will help someone in the future.

**Reply by mueslimak3r** [reply]
Exoplayer is the default because in most cases it is more stable. The main use-cases for libVLC is playing graphical subtitles without transcoding, and when exoplayer can't direct play for some reason.
